distsys/mpcalctx.go

```diff
@@ -206,17 +353,18 @@ func (ctx *MPCalContext) Abort() {
 		<-ch
 	}
 
-	ctx.dirtyResourceHandles = nil
+	// the go compiler optimizes this to a map clear operation
```

Comment:
yes, in principle.

old version:
```
m = null;
// ... we go start a new critical section
m = new HashMap<>();
```

new version:
```
m.clear(); // hopefully, this will avoid reallocating a good % of map data
```

(in pseudo-Java)

Reply:
Nice, didn't know that.